src/components/Navigation.js

```diff
@@ -1,35 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './navigation.css';
 import { NavLink } from 'react-router-dom';
+import Hamburger from 'hamburger-react';
 import logo from '../images/icons/logo.png';
 
-const Navigation = () => (
-  <div>
-    <nav className="navigation">
-      <section className="logo-div">
-        <img className="logo-image" src={logo} alt="Maths Magician Logo" />
-        Maths Magician
-      </section>
-      <section>
-        <ul className="nav-ul">
-          <li>
-            <NavLink className="nav-link" to="/">
-              Home
-            </NavLink>
-          </li>
-          <li>
-            <NavLink className="nav-link" to="/calculator">
-              Calculator
-            </NavLink>
-          </li>
-          <li>
-            <NavLink className="nav-link" to="/quotes">
-              Quotes
-            </NavLink>
-          </li>
-        </ul>
-      </section>
-    </nav>
-  </div>
-);
+function Navigation() {
+  const [isOpen, isClosed] = useState(false);
+  return (
+    <div>
+      <nav className="navigation">
+        <section className="logo-div">
+          <img className="logo-image" src={logo} alt="Maths Magician Logo" />
+          Maths Magician
+        </section>
+        <section className="for-phone">
+          <Hamburger
+            color="#fff"
+            duration={0.8}
+            toggled={isOpen}
+            toggle={isClosed}
+          />
+          {isOpen
+            ? (
+              <ul className="nav-ul-phone">
+                <li>
+                  <NavLink onClick={() => isClosed(false)} className="nav-link" to="/">
+                    Home
+                  </NavLink>
+                </li>
+                <li>
+                  <NavLink onClick={() => isClosed(false)} className="nav-link" to="/calculator">
+                    Calculator
+                  </NavLink>
+                </li>
+                <li>
+                  <NavLink onClick={() => isClosed(false)} className="nav-link" to="/quotes">
+                    Quotes
+                  </NavLink>
+                </li>
+              </ul>
+            ) : null}
```

Comment:
For this conditional rendering, I would like to suggest using  `Logical && rendering`, and the reason is, here we don't have any case to be executed for the false condition.
`{ isOpen && <Component> }`.

Also as all the links have the same style and only differ in the link value, I would like to suggest storing them in an array and generating them dynamically by running the `map` function on them. it will give your project more flexibility and adding new links would be super easy.